fsspec/implementations/local.py

```diff
@@ -16,6 +15,12 @@
 logger = logging.getLogger("fsspec.local")
 
 
+def _remove_prefix(text: str, prefix: str):
```

Comment:
Pretty soon we can use https://docs.python.org/3/library/stdtypes.html#str.removeprefix when we drop py 3.8